### DISCUSSION TITLE
Change key by existing key

### DISCUSSION THREAD

**Discussion by henriquelucas** [rustdesk/rustdesk#6914] [General]
I installed the Rustdesek server on Linux and it generated the key:
Key: mYtcJteQl1+WgKIIVSbRmpOkYxPrsSOBWYjfZeSj0S8= but I wanted to replace it with another one that already had Focn1GbvFJBfOWrHduUmrEifjVNtvVTFtc6QmQOt7TU= how can I make this replacement?

**Comment by xlionjuan** [comment]
Though it is public key, but you still shouldn't disclose your keys on the Internet..........

For your question, I think you could checkout this
https://rustdesk.com/docs/en/self-host/rustdesk-server-oss/install/
